ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -120,6 +120,12 @@ def ceil(x, name=None):
     return ivy.ceil(x)
 
 
+@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
```

Comment:
The original function's implementation https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/sinh_cn.html#sinh states that the supported 
dtypes are `float32`, `float64` and `float16`

Reply:
@AnnaTz  you mean  decorater of sinh should with_supported_dtype?

Reply:
Yes, exactly.